spec/fixtures/mockrequest.ts

```diff
@@ -85,6 +87,24 @@ export function generateIdToken(projectId: string): string {
   return jwt.sign(claims, mockKey.private_key, options);
 }
 
+/**
+ * Generates a mocked, unsigned Firebase ID token.
+ */
+export function generateUnsignedIdToken(projectId: string): string {
```

Comment:
Let's talk tomorrow. We could hypothetically build and parse invalid JWTs without the `jws` library. Right now, you're creating totally valid JWTs where the signature is signed with secret ''. If you wanted to create a JWT that just didn't have a valid signature you could do:

```
[
  {alg: 'HS256', typ: 'JWT'},
  { aud: projectId, sub: mokKey.user_id},
  "Invalid signature"
]
  .map(str => JSON.stringify(str))
  .map(str => Buffer.from(str).toString('base64'))
  .join('.')
```

And to parse without validating, we'd say:

```
JSON.parse(Buffer.from(token.split('.')[1], "base64").toString());
```

Reply:
This works!! Thanks for the suggestion - able to remove `jws` from our dep.